lib/views/customer_details.erb

```diff
@@ -29,6 +29,9 @@
             <div class="form-group">
               <label for="shipping-customer-name">Shipping Name<span class="text-danger">*</span></label>
               <input
+                <%if @errors[0]==:missing_shipping_customer_name%>
```

Comment:
Could wrap up the `@errors[0] == :missing_shipping_customer_name` into a helper method